test_notebooks/time_variable_Deff.py

```diff
@@ -0,0 +1,89 @@
+"""
+A script for calculating Deff based on MSD data from t=0 to a user-selected
+timepoint or range of timepoints (in which case Deff is a geometric mean).
+"""
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+import scipy
+from scipy import stats
+
+from bokeh.plotting import figure, show, gridplot
+import zipfile
+
+
+ALPHA = 1
+
+
+zf = zipfile.ZipFile('brain-diffusion_data.zip')
+file_handle = zf.open('Mean_Square_Displacement_Data.csv')
+msd = pd.read_csv(file_handle)
+# Rename columns without biological replicate number (this allows for
+# averaging of biological replicates later)
```

Comment:
An explanation that biological replicate is referring to brain sample could help here, instead of later.